scripts/remote_customize_entrypoint.sh

```diff
@@ -109,7 +90,8 @@ fi
 
```

Comment:
The return code check a few lines above (not part of this PR) has the same issue as I mentioned in [this other PR review comment](https://github.com/Cray-HPE/ims-kiwi-ng-opensuse-x86_64-builder/pull/97#discussion_r2363898767)